aviary/utils/process_input_decks.py

```diff
@@ -164,10 +167,58 @@ def update_dependent_options(aircraft_values: AviaryValues(), dependent_options)
 
 def initial_guessing(aircraft_values: AviaryValues()):
     problem_type = aircraft_values.get_val('problem_type')
+    reserves_option = aircraft_values.get_val(Aircraft.Design.RESERVES_OPTION)
     reserves = aircraft_values.get_val(
-        Aircraft.Design.RESERVES) if initial_guesses['reserves'] == 0 else initial_guesses['reserves']
+        Aircraft.Design.RESERVES, units='lbm') if initial_guesses['reserves'] == 0 else initial_guesses['reserves']
```

Comment:
the initial guessing wasn't intended to be used to set/override the actual value used in the reserve calculations, it was just intended to provide reasonable guesses for masses along the trajectory to improve the convergence without requiring users to set every point. If we stick with that, we don't need all this detailed evaluation of the options and error checking, because all that will be handled when FRESF is read in.

Reply:
Now, I keep the only GASP variable FRESF and let Fortran_to_Aviary to process the GASP .dat file.